ncloud/data_source_ncloud_nks_versions.go

```diff
@@ -0,0 +1,90 @@
+package ncloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/ncloud"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func init() {
+	RegisterDataSource("ncloud_nks_version", dataSourceNcloudNKSVersion())
+}
```

Comment:
```suggestion
func init() {
	RegisterDataSource("ncloud_nks_versions", dataSourceNcloudNKSVersions())
}
```

`data_source_ncloud_nks_versions` 리소스를 전체 조회하는 데이터소스면 이렇게 네이밍이 변경되어야할 것 같습니다.

Reply:
`data_source_ncloud_nks_versions` 로 변경하겠습니다.

Reply:
`ncloud_nks_version` -> `ncloud_nks_versions` 수정하겠습니다.